cockroachcloud/quickstart.md

```diff
@@ -65,7 +60,7 @@ You will see a welcome message when you've successfully connected to your cluste
 
 ## Step 3. Insert data
 
-1. You can now run [CockroachDB SQL statements](learn-cockroachdb-sql.html):
+1. In the SQL shell, run the following [SQL statements](learn-cockroachdb-sql.html):
```

Comment:
Let's skip directly to having the user access their cluster programmatically. Maybe we could include a separate bonus section on how to manually run SQL statements at the end.

Reply:
Done.

See https://github.com/cockroachdb/quickstart-code-samples/pull/3 for code changes.